src/ts/components/seekbar.ts

```diff
@@ -60,6 +60,13 @@ export interface SeekBarConfig extends ComponentConfig {
    * Used to enable/disable seek preview
    */
   enableSeekPreview?: boolean;
+
+  /**
+   * Enables a fallback mode for handling ARIA details of the seekbar for platforms that don't support the 
+   * `aria-valuetext` attribute, such as Samsung Tizen. In this mode, the `aria-label` attribute is used in addition
+   * to provide the current time.
+   */
+  ariaFallbackMode?: boolean;
```

Comment:
The variable name kind of sounds like you could configure a `mode`, but it's only a flag - I would suggest to name it `enableAriaFallbackMode` or so

Reply:
I've renamed it to 919bce2423100e248459fc7694f9201930ce6a59, hopefully this makes it clearer.